adds initial rough outline (#1)
learn-co-curriculum/prework-videos-intro-to-hashes: README.md

# Intro to Hashes

## Learning Goals

> + Explain when you would want to use a hash to store data

Reviewer: We could make this learning goal a bit stronger by being explicit about the reason/s.

Author: ```suggestion
+ Explain that hashes are useful to store unordered data
```
